### PR TITLE
feat(quic): multi streams

### PR DESCRIPTION
**Pull request by qzhuyan**
## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/<version>/(feat|fix)-<PR-id>.en.md` and `.zh.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [x] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
EMQX-8912
- [x] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx/src/emqx_connection.erl [hunk 1]
```diff
@@ -14,7 +14,12 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
-%% MQTT/TCP|TLS Connection
+%% This module interacts with the transport layer of MQTT
+%% Transport:
+%%   - TCP connection
+%%   - TCP/TLS connection
+%%   - WebSocket
```
Comment (HJianBo) [inline apps/emqx/src/emqx_connection.erl:21]: Websocket conns is managed by the `emqx_ws_connection` module.

File: apps/emqx/src/emqx_connection.erl [hunk 2]
```diff
@@ -111,7 +116,10 @@
     limiter_buffer :: queue:queue(pending_req()),
 
     %% limiter timers
-    limiter_timer :: undefined | reference()
+    limiter_timer :: undefined | reference(),
+
+    %% QUIC conn owner pid if in use.
+    quic_conn_pid :: maybe(pid())
```
Comment (HJianBo) [inline apps/emqx/src/emqx_connection.erl:122]: Do we need concern about the Hot-upgrade compatibility once e5.0.0 released?

Comment (qzhuyan) [reply apps/emqx/src/emqx_connection.erl:122]: I checked with @id , We don't support hot upgrade for e5.0.x

File: changes/v5.0.17/feat-9949.en.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+QUIC transport Multistreams support and QUIC TLS cacert support.
```
Comment (ieQu1) [inline changes/v5.0.17/feat-9949.en.md:1]: The location of the changelog fragment has been changed, I think.

**Review by zmstone — APPROVED** [review]

**Pull request merged**